synopsys-task/src/synopsys-task/tools-parameter.ts

```diff
@@ -80,14 +80,44 @@ export class SynopsysToolsParameter {
       polData.data.polaris.triage = inputs.POLARIS_TRIAGE;
     }
 
+    if (parseToBoolean(inputs.POLARIS_PR_COMMENT_ENABLED)) {
+      console.info("Polaris PR comment is enabled");
+      if (!inputs.AZURE_TOKEN) {
+        throw new Error(
+          "Missing required azure token for pull request comment"
+        );
+      }
+
+      polData.data.azure = this.setAzureData(
+        "",
```

Comment:
Instead of setting blank values for all can we set only azure token?

Reply:
As we discussed, while working on other platforms' simplification, we will remove entire structure excluding azure token